aframe/pipelines/sandbox/sandbox.cfg

```diff
@@ -95,7 +95,7 @@ request_cpus = 1
 
 [luigi_TimeslideWaveforms]
 workflow = htcondor
-Tb = &::luigi_base::Tb
+num_injections = 100000
```

Comment:
We should play around with how many injections we actually need (which this PR will make a lot easier to do). When we fill a year of timeslides, we get O(1e6) injections, but also our error bars are quite tight already.

Reply:
Yeah right - we can probably get away with fewer